src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -33,7 +33,10 @@
       return Collections.emptyList();
     }
     return Collections.singletonList(
-        OutputSpec.newBuilder().addPrint("%s").addPrint(RESPONSE_PLACEHOLDER).build());
+        OutputSpec.newBuilder()
```

Comment:
I'm confused what this is doing. Is it so that we don't create any output here, and use the template logic to write `print(response)`. In that case, I think we should delete this method altogether.

Reply:
Not quite. The result of this, with the template I have, is that we get a `print(response)` rather than `print('{}'.format(response))`. The former is more idiomatic if we're not formatting.

Reply:
(BTW, this only works if no `OutputSpec` is specified at all, which is when we call this. It seems we can't pass an empty string explicitly as the first `print` list element; the yaml-to-proto process seems to barf.)

Reply:
As discussed offline, changed so that the view model still corresponds to `%s`. The logic for making the output more idiomatic will be contained in the `.snip` file.